src/test/java/mate/academy/hibernate/relations/dao/impl/MovieDaoImplTest.java

```diff
@@ -107,4 +111,13 @@ private void verifyCreateMovieWorks(MovieDao movieDao, Movie movie, Long expecte
         Assert.assertEquals(expectedId, actual.getId());
         Assert.assertEquals(movie.getTitle(), actual.getTitle());
     }
+
+    private void initializeActors(Movie movie){
+        try(Session session = getSessionFactory().openSession()){
+            session.refresh(movie);
+            Hibernate.initialize(movie.getActors());
```

Comment:
`Hibernate.initialize` is not the best approach to fix LazyInitializationException. Can we use join fetch here?